tests/env/test_create.py

```diff
@@ -105,25 +105,59 @@ def test_create_advanced_pip(
     tmp_envs_dir: Path,
     support_file_isolated,
 ):
-    env_name = uuid4().hex[:8]
-    prefix = tmp_envs_dir / env_name
+    import shutil
+    import subprocess
 
-    # avoid pip touching support_file paths inside source checkout
-    environment_yml = support_file_isolated(Path("advanced-pip", "environment.yml"))
+    # Get the argh directory path
+    argh_dir = Path(support_file("advanced-pip/argh"))
```

Comment:
so support_file_isolated("advanced-pip/argh") hopefully finds argh in a temporary directory that we don't have to clean up.